src/3k-mapper.xml

```diff
@@ -1154,6 +1169,170 @@ end</script>
                     <packageName></packageName>
                     <regex>^tmap addalias &quot;(.+)&quot; (.+)$</regex>
                 </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Rename an area</name>
+                    <script>--renames current area
+local Name = matches[2]
+local cArea = getRoomArea(tmap.lastId)
+setAreaName( cArea, Name )</script>
```

Comment:
With the enforced uniqueness of area names, `setAreaName( oldArea, newArea )` now accepts an existing area name as a string for `oldArea` BTW...